addons/web/static/src/js/views/basic/basic_model.js

```diff
@@ -1128,6 +1128,9 @@ var BasicModel = AbstractModel.extend({
                                 _.extend(record.data, _changes);
                                 resolve(changedFields);
                             }
+                            if (record.model === 'res.company') {
+                                self.do_action('reload_context');
+                            }
```

Comment:
I'd do it in _invalidateCache. Also, a test would be nice.

Reply:
Thanks. The test will come with: https://www.odoo.com/web#id=2188789&action=333&active_id=1251&model=project.task&view_type=form&menu_id=4720